### PR TITLE
[PATCH v4] example/test/validation: remove usage of deprecated timer functions

### PR DESCRIPTION
**Pull request by MatiasElo**
Remove usage of deprecated API functions `odp_timer_set_abs()` and `odp_timer_set_rel()`.

### CODE REVIEW

**Comment by MatiasElo** [comment]
V2:
- Fixed typos in commit messages

**Comment by MatiasElo** [comment]
V3:
- Removed references also from user's guide

File: doc/users-guide/users-guide-timer.adoc [hunk 1]
```diff
@@ -46,12 +46,12 @@ as an input parameter to enable the pool-specific conversion ratios to be
...
 
 Associated with each timer pool is a free running tick counter that can be
-sampled at any time via the `odp_timer_current_tick()` API. Timers can be set
-to an absolute future tick value via `odp_timer_set_abs()` or to a future tick
-value relative to the current tick via `odp_timer_set_rel()`.  Implementations
-may impose minimum and maximum future values supported by a given timer pool
-and timer set operations will fail if the requested value is outside of the
-supported range.
+sampled at any time via the `odp_timer_current_tick()` API. Timers are started
+with `odp_timer_start()` API and the expiration time can be an absolute
```
Comment (JereLeppanen) [inline doc/users-guide/users-guide-timer.adoc:50]: I would just drop the word "API" on this line. It's obvious that it's an API function.

**Review by JereLeppanen — APPROVED** [review]

File: doc/users-guide/users-guide-timer.adoc (hunk 1 above)
Comment (MatiasElo) [reply doc/users-guide/users-guide-timer.adoc]: Dropped in V4.

**Review by psavol — APPROVED** [review]

**Pull request merged**